packages/contracts/contracts/universal/MultiSigWallet.sol

```diff
@@ -16,6 +16,11 @@ import { IMultiSigWallet } from "./IMultiSigWallet.sol";
  *         Allows multiple parties to agree on transactions before execution.
  */
 abstract contract MultiSigWallet is IMultiSigWallet, ReentrancyGuardUpgradeable {
+    /**
+     * @notice The address of the governor contract. Can be updated via upgrade.
```

Comment:
Why don't we add some comments here about governor's timelock contract?
Would it be okay without any further comments?

Reply:
From a multi-sig wallet's perspective, I think it's right to say that it don't know if Governor uses time-lock or not.

Reply:
I see!